jobs/JGDAS_GLOBAL_OCEAN_ANALYSIS_PREP_OBS

```diff
@@ -0,0 +1,43 @@
+#!/bin/bash
+export STRICT="NO"
```

Comment:
Why is this NO?

Reply:
We have an outstanding issue with variables not getting assigned (I suspect because we don't use them) - this is being worked on

Reply:
https://github.com/NOAA-EMC/global-workflow/issues/1837